features/general/file-manager.md

```diff
@@ -0,0 +1,105 @@
+# File Manager of ERP.net
+
+## Overview
+
+Oftentimes users have to search through complex directories to find a file they are looking for. 
+Especially when the file is **not associated** to a **specific document or record** in the ERP system, buth is a rather more **generic document** for business use, such as documentation regarding training courses, internal or external regulations, company practices, handbooks, and so on.  
```

Comment:
This says what it is NOT. However, someone might need specifically to attach file to a document and might be repulsed by this description. Generally, never use negative connotation.

It might be better to explain, that File Manager build upon the proven and trusted functionality of Files, attached to documents and other objects. And now you can just upload an unattached file. This way, users will know that they have BOTH functionalities baked in erp.net.

Reply:
All traces that can be regarded as negative connotation were removed and replaced by a more empowering style of writing.